website/docs/r/event_orchestration_path_resource.html.markdown

```diff
@@ -0,0 +1,222 @@
+* `event_action` - (Optional) sets whether the resulting alert status is trigger or resolve. Allowed values are: `trigger`, `resolve`
+* `variables` - (Optional) Populate variables from event payloads and use those variables in other event actions.
+  * `name` - (Required) The name of the variable
+  * `path` - (Required) Path to a field in an event, in dot-notation. This supports both [PD-CEF](https://support.pagerduty.com/docs/pd-cef) and non-CEF fields. Eg: Use `event.summary` for the `summary` CEF field. Use raw_event.fieldname to read from the original event `fieldname` data.
+  * `type` - (Required) Only `regex` is supported
+  * `value` - (Required) The Regex expression to match against. Must use valid [RE2 regular expression](https://github.com/google/re2/wiki/Syntax) syntax.
+* `extractions` - (Optional) Replace any CEF field or Custom Details object field using custom variables.
+  * `template` - (Optional) A value that will be used to populate the `target` field. The configuration can include variables extracted from the payload by using string interpolation. Eg: If you have defined a variable called `hostname` you can set extraction `template` to `High CPU on variables.hostname server` to use the variable in extraction.  This field can be ignored for `regex` based replacements.
+  * `target` - (Required) The PagerDuty Common Event Format [PD-CEF](https://support.pagerduty.com/docs/pd-cef) field that will be set with the value from the `template` or based on `regex` and `source` fields.
+  * `regex` - (Optional) The conditions that need to be met for the extraction to happen. Must use valid [RE2 regular expression](https://github.com/google/re2/wiki/Syntax) syntax. This field can be ignored for `template` based replacements.
+  * `source` - (Optional) Field where the data is being copied from. Must be a PagerDuty Common Event Format [PD-CEF](https://support.pagerduty.com/docs/pd-cef) field. This field can be ignored for `template` based replacements.
+
+### Catch All (`catch_all`) supports the following:
+* `actions` - (Required) These are the actions that will be taken to change the resulting alert and incident. `catch_all` supports all actions described above for rules except `route_to` action.
+
+## Attributes Reference
+
+The following attributes are exported:
+* `parent`
+  * `type` - Type of the parent (Event Orchestration) reference for this Event .Orchestration Path
+  * `self` - The URL at which the parent object (Event Orchestration) is accessible.
+* `self` - The URL at which the Unrouted Event Orchestration path is accessible.
+* `rules`
+  * `id` - The ID of the rule within the `start` set.
+* `catch_all`
+  * `actions`
+    * `suppress` - The suppress action for catch_all rule. This is always True.
+
+## Import
+
+Unrouted Orchestration path can be imported using the `id` of the event orchestration, e.g.
+
+```
+$ terraform import pagerduty_event_orchestration_unrouted 1b49abe7-26db-4439-a715-c6d883acfb3e
```

Comment:
We are yet to test out imports. This command might require a change based on what we find during QA.